nautobot/docs/installation/centos.md

```diff
@@ -156,7 +156,7 @@ owners.
 
 Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
 
-mysql> CREATE DATABASE nautobot CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;
+mysql> CREATE DATABASE nautobot;
```

Comment:
Is this removed because mysql defaults to the correct character set and collation if you don't specify anything? Have we confirmed this on centos specifically?

Reply:
Yes on both counts.